bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3416,7 +3416,9 @@ spec:
     Drivers\n*\tResiliency - Designed to make Kubernetes applications more resilient
     to \n  node failures\n\nThese modules are available for various storage platforms
     in the Dell portfolio, \nsuch as Dell PowerStore, Dell PowerMax, Dell PowerFlex,
-    Dell PowerScale, and Dell Unity XT.\n\nFor more information, visit [Dell’s Container
+    Dell PowerScale, and Dell Unity XT.\n\n### Supported Openshift Platforms\n\nQualified Openshift versions for CSM Operator v1.6.0: 4.15-4.16
```

Comment:
Do we need to update Operator versions here or not since this is a patch?

Reply:
looks like this is the first patch to include this line.
Since it's 1.6.0 was fully qualified, I'll leave it as is